### PR TITLE
fix: #767

### PR DESCRIPTION
**Pull request by wadouk**

### CODE REVIEW

File: analysers/analyser_merge_defibrillators_FR_gers.py [hunk 1]
```diff
@@ -32,8 +32,9 @@ def __init__(self, config, logger = None):
         self.init(
             u"https://www.data.gouv.fr/fr/datasets/inventaire-des-defibrillateurs-automatises-externes-dae-dans-le-gers/#",
             u"Inventaire des Défibrillateurs Automatisés Externes (DAE) dans le Gers",
-            GeoJSON(Source(attribution = u"Région Occitanie / Pyrénées Méditerranée",
-                    fileUrl = u"https://www.data.gouv.fr/fr/datasets/r/cc36c26e-e782-47bf-8fd6-a03aa459e8d0")),
+            SHP(Source(attribution = u"Région Occitanie / Pyrénées Méditerranée",
+                    fileUrl = u"https://www.data.gouv.fr/fr/datasets/r/d81ffeca-921b-4455-b889-f1019da423e1",
+                    zip = "inventaire-des-defibrillateurs-automatises-externes-dans-le-gers.shp")),
             Load("geom_x", "geom_y"),
```
Comment (frodrigo) [inline analysers/analyser_merge_defibrillators_FR_gers.py]: Cannot work. It is cood for GeoJson, not Shapefile.

```
Load(("ST_X(ST_Centroid(geom))",), ("ST_Y(ST_Centroid(geom))",), srid = 2154),
```

Check the SRID.

Comment (wadouk) [reply analysers/analyser_merge_defibrillators_FR_gers.py]: oups, of course

**Comment by frodrigo** [comment]
From Travis  TEST_SUITE=merge

```
E       NameError: name 'SHP' is not defined
```

**Comment by wadouk** [comment]
is it better ?

**Comment by frodrigo** [comment]
All right. Thank you.

**Pull request merged**